app/exerlog/lib/UI/login_screen/login_page.dart

```diff
@@ -1,33 +1,29 @@
-import 'package:exerlog/Bloc/authentication.dart';
 import 'package:exerlog/Bloc/email_signup.dart';
-import 'package:exerlog/Bloc/user_bloc.dart';
-import 'package:exerlog/Bloc/workout_bloc.dart';
-import 'package:exerlog/Models/exercise.dart';
-import 'package:exerlog/Models/sets.dart';
-import 'package:exerlog/Models/user.dart';
-import 'package:exerlog/Models/workout.dart';
-import 'package:exerlog/UI/login_screen/google_signin_button.dart';
+import 'package:exerlog/UI/global.dart';
+import 'package:exerlog/UI/gradient_button.dart';
 import 'package:exerlog/UI/login_screen/login_data.dart';
 import 'package:exerlog/UI/login_screen/login_form.dart';
 import 'package:exerlog/UI/login_screen/signup_form.dart';
-import 'package:exerlog/UI/workout/workout_builder.dart';
 import 'package:exerlog/UI/workout/workout_page.dart';
 import 'package:exerlog/main.dart';
-import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter/material.dart';
-import '../gradient_button.dart';
-import '../global.dart';
-import '../maxes/max_builder.dart';
 
 class LoginPage extends StatefulWidget {
-  LoginPage(this.title, {Key? key}) : super(key: key);
-  String title = "0";
+  final String title;
+
+  const LoginPage({
+    Key? key,
+    this.title = "0",

```

Comment:
With this change an error occurs in the file 'main.dart' one line 60. 
The value `title` is a named parameter. Change in 'main.dart' from `LoginPage('1')` to `LoginPage(title:'1')` or `LoginPage()`

Reply:
Forgot to add the file to the commit, thanks